contracts/ReaperAutoCompoundScreamLeverage.sol

```diff
@@ -230,36 +229,41 @@ contract ReaperAutoCompoundScreamLeverage is ReaperBaseStrategy {
         _onlyStrategistOrOwner();
         withdrawSlippageTolerance = _withdrawSlippageTolerance;
     }
+
+    /**
+     * @dev Sets the swap path to go from {WFTM} to {want}.
+     */
+    function setWftmToWantRoute(address[] calldata _newWftmToWantRoute) external {
+        _onlyStrategistOrOwner();
+        require(_newWftmToWantRoute[0] == WFTM, "bad route");
+        require(_newWftmToWantRoute[_newWftmToWantRoute.length - 1] == want, "bad route");
+        delete wftmToWantRoute;
+        wftmToWantRoute = _newWftmToWantRoute;
```

Comment:
Isn't re-assigning the variable also deleting the old one? Why the need to delete?

Reply:
Probably. Just being a bit paranoid about it.